### PR TITLE
Split codebase into packages (workspaces)

### PR DESCRIPTION
**Pull request by marcperez**
## Summary
This PR splits the codebase into 3 packages: `adyen-web`, `playground` and `server` by leveraging Yarn Workspaces.

### CODE REVIEW

File: packages/playground/src/pages/Dropin/Dropin.js [hunk 1]
```diff
@@ -1,4 +1,5 @@
-import AdyenCheckout from '../../../src';
+import AdyenCheckout from '@adyen/adyen-web';
+import '@adyen/adyen-web/dist/adyen.css';
```
Comment (pabloai) [inline packages/playground/src/pages/Dropin/Dropin.js:2]: Why do we only load this CSS here and not in the other pages?

File: packages/playground/src/style/border-radius.scss [hunk 2]
```diff
@@ -0,0 +1,4 @@
+$border-radius-small: 3px;
```
Comment (pabloai) [inline packages/playground/src/style/border-radius.scss:1]: I wouldn't duplicate all these files. If we cannot access them from the playground, that's fine, let's just remove them and not use these variables.
That being said, feel free to just merge this PR and then we can remove them in another PR.

File: packages/playground/src/pages/Dropin/Dropin.js (hunk 1 above)
Comment (marcperez) [reply packages/playground/src/pages/Dropin/Dropin.js:2]: That's fixed now 👍

File: packages/playground/src/style/border-radius.scss (hunk 2 above)
Comment (marcperez) [reply packages/playground/src/style/border-radius.scss:1]: Fair point! I think it'd be better to refactor this in a separate PR, not to add even more changes to this one :)

File: packages/playground/src/style.scss [hunk 3]
```diff
@@ -1,4 +1,4 @@
-@import '../src/style';
+@import './style/index';
```
Comment (pabloai) [inline packages/playground/src/style.scss:1]:
```suggestion
@import './style';
```

**Review by pabloai — APPROVED** [review]

**Pull request merged**